src/track.jsx

```diff
@@ -68,7 +67,7 @@ var renderSlides = (spec) => {
     }
 
     var childStyle = getSlideStyle(assign({}, spec, {index: index}));
-    slides.push(cloneWithProps(child, {
```

Comment:
So if each slide can't share props with the ability to override props, the docs you point me to say it should use `React.addons.cloneWithProps`. Is that what you mean? The slideshow does work with what I have it the pull request.